Aspire Budgeting/Facilities/AspireFonts.swift

```diff
@@ -24,4 +24,8 @@ extension Font {
   static func nunitoSemiBold(size: CGFloat) -> Font {
     return Font.custom("Nunito-SemiBold", size: size)
   }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/080f65df2311b45c8d4b592dff399d3b1bb01b5f/Aspire Budgeting/Facilities/AspireFonts.swift#L27"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AspireFonts.swift:27</code></td>
    </tr>
  </tbody>
</table>